packages/server/src/fhir/routes.ts

```diff
@@ -71,7 +71,23 @@ fhirRouter.use((req: Request, res: Response, next: NextFunction) => {
       res.contentType(ContentType.FHIR_JSON);
     }
 
-    return res.json(data);
+    let legacyFhirJsonResponseFormat: boolean | undefined;
+    try {
+      const ctx = getAuthenticatedContext();
+      legacyFhirJsonResponseFormat = ctx.project.systemSetting?.find(
+        (s) => s.name === 'legacyFhirJsonResponseFormat'
+      )?.valueBoolean;
+    } catch (_err) {
```

Comment:
Why do we ignore errors here? This is just because we are not guaranteed to have an authenticated context here?

Reply:
Yep, there were a couple tests that pass through this while unauthenticated. I'll update the comment to clairify.